platform/ecs/task/task.go

```diff
@@ -180,13 +181,13 @@ func getDockerID(proc procfs.Proc) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	memCgroup := cgroup["memory"]
+	memCgroup := cgroup[memorySubsystem]
 	if memCgroup == nil {
 		return "", errors.New("memory cgroup not exists")
 	}
 	parts := strings.Split(memCgroup.CgroupPath, string(os.PathSeparator))
-	if len(parts) != 4 || parts[1] != "ecs" {
-		return "", fmt.Errorf("faild to parse %s", memCgroup.CgroupPath)
+	if parts[1] != "ecs" {
```

Comment:
This might not happen but it's better to check `len(parts) > 2` to avoid index crash.

Reply:
Thank you. I'll fix it.